types/api/addressParams.ts

```diff
@@ -1,3 +1,4 @@
+import type { AddressFilecoinParams } from './address';
```

Comment:
It might be better to keep this type in this file, as it is a sub-modal of the `AddressParam`.